src/preprocess.jl

```diff
@@ -0,0 +1,35 @@
+"""
+    preprocess(::File, actual; kw...) -> out
+
+Preprocess `actual` so that `out` is similar to reference file content.
+"""
+preprocess(::File, actual; kw...) = actual
```

Comment:
Can we rename to to something more clear, like `convert_towards_ format`.
And maybe rather than taking the `File` just take the `Format` type

Reply:
renamed to `_convert` in https://github.com/Evizero/ReferenceTests.jl/pull/37/commits/8bb84a23c3776995332c7cf6091ce1bb1d53b5d9